hw/formal/fpv_vcf.tcl

```diff
@@ -0,0 +1,164 @@
+fvdisable {*spi_device.u_reg.u_socket.tlul_assert_host.responseSizeMustEqualReq*}
+fvdisable {*spi_device.tlul_assert_host.responseSizeMustEqualReq*}
+
+# hmac
+fvdisable {*hmac.u_tlul_adapter.tlul_assert_host.onlyOnePendingReq*}
+fvdisable {*hmac.u_reg.u_socket.tlul_assert_device.gen_assert[0]*onlyOnePendingReq*}
+
+# flash_ctrl
+fvdisable {*flash_ctrl.tlul_assert_host.response*Must*}
+fvdisable {*flash_ctrl.u_reg.u_socket.tlul_assert_*.response*Must*}
+fvdisable {*flash_ctrl.u_reg.u_socket.tlul_assert_device.gen_assert*.tlul_assert.onlyOnePendingReq*}
+
+# xbar
+fvdisable {*xbar_main.tlul_assert_device_*.sizeMatches*}
+fvdisable {*xbar_main.tlul_assert_device_*.legalA*}
+fvdisable {*xbar_main.tlul_assert_device_*.addressAligned*}
+fvdisable {*xbar_main.tlul_assert_device_*.maskMustBeCont*}
+fvdisable {*xbar_main.tlul_assert_host_*.legal*}
+fvdisable {*xbar_main.u_*.tlul_assert_host.legalDParam*}
+fvdisable {*xbar_main.u_*.tlul_assert_device.response*}
+fvdisable {*xbar_main.u_*.tlul_assert_device.legalA*}
+fvdisable {*xbar_main.u_*.tlul_assert_device.addressAligned*}
+fvdisable {*xbar_main.u_*.tlul_assert_device.checkResponseOp*}
+fvdisable {*xbar_main.u_*.tlul_assert_device.maskMustBeCont*}
+fvdisable {*xbar_main.u_*.tlul_assert_device.sizeMatches*}
+fvdisable {*xbar_main.u_*.tlul_assert_*.gen_assert*.tlul_assert.maskMustBeCont*}
+fvdisable {*xbar_main.u_*.tlul_assert_*.gen_assert*.tlul_assert.addressAlignedToSize*}
+fvdisable {*xbar_main.u_*.tlul_assert_*.gen_assert*.tlul_assert.legal*}
+fvdisable {*xbar_main.u_*.tlul_assert_*.gen_assert*.tlul_assert.sizeMatches*}
+
+# top_earlgrey
+fvdisable {top_earlgrey.*addressAligned*}
+fvdisable {top_earlgrey.*tlul_assert*Must*}
+fvdisable {top_earlgrey.*onlyOne*}
+fvdisable {top_earlgrey.*Response*}
+fvdisable {top_earlgrey.*legal*}
+fvdisable {top_earlgrey.u_xbar_main.u_sm1_*.rspIdInRange}
+fvdisable {top_earlgrey.u_xbar_main.*depthShall*}
+fvdisable {top_earlgrey.u_xbar_main.*tlul_assert*DataKnown*}
+fvdisable {top_earlgrey.u_dm_top.*tlul_assert_*DataKnown*}
+
+#-------------------------------------------------------------------------
+# Configure grid usage
+#-------------------------------------------------------------------------
+# Use "set_grid_usage" option to launch the run on the grid. 
+# Use option "-type <LSF|RTDA|SGE>=<# of workers>" to specify the type of grid and the numbers of workers
+# "-control {<submission commands>} is used to specify the exact qsub/bsub string to use for accessing the grid resources 
+# Ex: set_grid_usage -type sge=12 -control { qsub -P <machine name> } 
+set_grid_usage -type RSH=12
+
+#-------------------------------------------------------------------------
+# Run all the assertion and cover properties 
+#-------------------------------------------------------------------------
+# time limit set to 2 hours
+set_fml_var fml_max_time 2H
+
+# initialize the design 
+sim_run -stable
+sim_save_reset
+
+check_fv -block
```

Comment:
I am not familiar with VC Formal, could you elaborate more about the usage of these three lines, just out of my curiosity.